components/micropython/Makefile

```diff
@@ -46,6 +46,10 @@ CFLAGS += \
 LDFLAGS += -L$(BOARD_DIR)/lib -L$(SDDF)/lib -L$(LIBMATH)
 LIBS += -lm -lmicrokit -Tmicrokit.ld -lc
 
+ifdef EXEC_MODULE
+CFLAGS += -DEXEC_MODULE=\"$(EXEC_MODULE)\"
+endif
```

Comment:
Minor request, can you put this above the `ENABLE_I2C` check just to group all of our custom configuration together.